### ISSUE TITLE
Good fix? Redirect blocked by ad blockers.

### ISSUE THREAD

**Issue by elvey** [troyhunt/Coinhive#12]
Hi, @troyhunt & friends! 

1️⃣GOOD IDEA?
My question is about coinhive (this project, that is) and ad blockers. 

I propose we ask to have coinhive.com removed from ad blocker lists.   I wonder if we should, and want to see if there's consensus first. Would it be good to reach more folks?  Make the alert appear more consistently rather than on on users machines, only to not be reproducible on the machines of admins more likely to run ad blockers?  I think it probably would.  

2️⃣HOW?
In the case of the ad blocker I use⚚, it was added 5 years ago: [github.com/dgraham/Ka-Block/blame/... /blockerList.json#L5218](https://github.com/dgraham/Ka-Block/blame/f2bfbf1ce71ee24e30a572e4f3cdf387b3adfc62/Shared%20(Extension)/blockerList.json#L5218) 
They say to [open a new issue here](https://github.com/dgraham/Ka-Block/issues/new/choose) to request changes. 

**But the main thing is __easylist__, which is way more popular**. It contains (in its sub-list easyprivacy's trackingservers sub-sub-list)
`/coinhive.min.js` and I guess needs to continue to contain it.   But the maintainers could be asked to remove
`||coinhive.com^$third-party`.  They ask folks to start at [the easylist forums](https://forums.lanik.us/).  
(So I'm not @ pinging the [folks who added it or last edited the lists](https://github.com/easylist/easylist/blob/ca94abc614e79694eae97180f09e516e65dc504a/easyprivacy/easyprivacy_trackingservers.txt#L2989) .
<hr>
⚚  I ran into this by trying to create an account at http://forum.oldversion.com/forum.php - but only because I happened to turn off content blockers on the site.  (I have used oldversion.com on occasion but the forum is a ghost town.) (And I've enjoyed your project write-ups many times, but hadn't seen this one.). 

My claim to fame was [an (almost successful?) legal effort to get brokers to submit to public security audits](https://caringaboutsecurity.wordpress.com).  FYI. 

Also I wonder how many hits you get to other parts of the site like https://coinhive.com/documentation/authedmine.  Interesting to [see what it once looked like](https://archive.fo/vd4Cx) - someone was whining about it being added to easylist.

**Comment by iam-py-test** [comment]
Does `@@||coinhive.com^$all` work?
Also, I have reached out to the maintainers of various filterlists to have them unblock it (i.e https://github.com/tcptomato/ROad-Block/pull/118)
> /coinhive.min.js

There may be some website which self-host the entire thing (maybe?)

**Comment by elvey** [comment]
I have/had not taken action because I had received no response to my question - "GOOD IDEA? ..." from @troyhunt or anyone.

I suppose your comment is an implicit answer: Yes.  And based on your self-description and commit history, you know what you're talking about.

Your PR,  tcptomato/ROad-Block#118 is cool - so well documented.  If my creating this gave you the idea or drive, I'm happy.

( I don't really know enough to judge if it was appropriate but I'm surprised by the choice to move it to annoyances: 
https://github.com/easylist/easylist/commit/8c5218b001e374b107c88fbfb3fe37e64bbb37d3#diff-660ac35bad3047b69417687a0169bea3b9f93ff1710509c7c0fa63deea7f109e )

>Does @@||coinhive.com^$all work?
Surely.  But Ka-Block has no knobs.  And I don't want to manually whitelist it anyway.  

I guess we can close this as resolved by your actions?

**Comment by tcptomato** [comment]
I've merged your PR.  So I think you can close this ticket.

**Comment by iam-py-test** [comment]
>  So I think you can close this ticket.

I think there are other lists which block it though

**Comment by elvey** [comment]
Yeah, CoinhiveDOTcom is still in NOCOIN - https://github.com/hoshsadiq/adblock-nocoin-list/blob/master/nocoin.txt, which @smed79 maintains and which is "Included" or "Active" by default in 5+ ad blockers. 

 (I'd guess one can't @ people not in the project, but I'm trying anyway.)

**Comment by iam-py-test** [comment]
> (I'd guess one can't @ people not in the project, but I'm trying anyway.)

It should work...

**Comment by smed79** [comment]
@elvey 
> CoinhiveDOTcom is still in NOCOIN

Removed from **Nocoin list** https://github.com/hoshsadiq/adblock-nocoin-list/commit/b1e4551039940b4b859963bb9b4b2410853d3305 & **Liste FR** https://github.com/easylist/listefr/commit/5f234115d600c1ee9b2573aa4049f9c23bf6fe15 
Also submitted a PR for **EasyList**  https://github.com/easylist/easylist/pull/11235

@ryanbr @Khrin fyi.

**Comment by ghost** [comment]
@pgl

**Comment by elvey** [comment]
RESOLVED!  

OP: Woah there!  Hold on-I see no evidence that Troy Hunt has control of coin-hive.com.  The registration information is quite different than that for coinhive.com. Yes they _currently_ redirect to the same site but that’s not good enough reason to delist the latter, I think  not yet.

  PS I don’t think Troy has visited this thread yet so we probably need to contact him another way than @troyhunt. 

PPS @smed79 - maybe pause your pull requests or explain? 

PPPS I am emailing Troy as that is how he prefers to be contacted. 

PPPPS: Troy responded, below.

**Comment by ghost** [comment]
@troyhunt Do you own `coin-hive.com`?

**Comment by iam-py-test** [comment]
@troyhunt can you clarify which ones you do/don't own (sorry for the redundant comment, didn't see https://github.com/troyhunt/Coinhive/issues/12#issuecomment-1064655856 before posting)

**Comment by PerfectSalad** [comment]
@aymeshkov @zloyden

**Comment by ghost** [comment]
@ameshkov @zloyden

**Comment by elvey** [comment]
I don’t think Troy has visited this thread yet so we probably need to contact him another way than @troyhunt.

 @smed79 - maybe pause your pull requests or explain?

 I am emailing Troy as that is how he prefers to be contacted, per his site.

**Comment by iam-py-test** [comment]
I also ping-ed him on Twitter

**Comment by ryanbr** [comment]
https://coinhive.com/lib/coinhive.min.js & https://coinhive.com/lib/coinhive.js  seems fine, no privacy issues or coin miner. Was removed already # edit.  With the rest of coinhive, coinhiveproxy.com is a "dead domain" is there any sites using it? also `/coinhive-proxy-`

**Comment by troyhunt** [comment]
Sorry folks, for some reason I got no alerts on this thread until someone on Twitter just pointed me to it, I'm not (deliberately) ignoring you!

Here's all the "coin" domains I own:

![image](https://user-images.githubusercontent.com/273244/157785596-0b1a33a4-6ab3-41a8-a5a7-377f158e3f21.png)

So yes, I do own coin-hive.com and am redirecting it to my blog post on Coinhive.

I'm supportive of removing it from ad blockers simply to raise awareness of which sites currently have it embedded. That's obviously not my decision though and I don't know that it would really make a lot of difference as the majority of people visiting those sites won't have blockers anyway. But yeah, the more exposure the better 🙂

**Comment by pgl** [comment]
Hey. `coin-hive.com` was still on my list (added Mon Dec 11th 2017 21:53).

I agree that it should be removed, so it's gone now.

**Comment by zloyden** [comment]
Hi! I have removed it from AdGuard filters.
